execution_workflows/GETUTR/bin/postprocess_relative_quantification.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+
+
+def parse_args(args=None):
+    description = "Reformat GETUTR output bed file into the output file of relative usage quantification challenges"
+    epilog = "Example usage: python convert_to_bed.py <FILE_IN> <RUN_IDENTIFICATION> <RELATIVE_USAGE_QUANTIFICATION_OUT>"
```

Comment:
the parser only has two arguments. I guess <RUN_IDENTIFICATION> is not needed, correct?

Reply:
you're right!